workflows/proteomics/openms-metaprosip/.dockstore.yml

```
@@ -0,0 +1,7 @@
+version: 0.1
+workflows:
+- name: main
+  primaryDescriptorPath: /metaprosip.ga
+  subclass: Galaxy
+  testParameterFiles:
+  - /metaprosip-tests.yml
```

```suggestion
  - /metaprosip-tests.yml
  authors:
  - name: Matthias Bernt
    orcid: 0000-0003-3763-0797
```

(this should be autogenerated by planemo too, hopefully)